fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -816,6 +820,16 @@ public static FHIRSearchContext parseQueryParameters(Class<?> resourceType,
             }
         } // end for
 
+        try {
+            // Check for valid search parameter combinations
+            checkSearchParameterCombinations(resourceType, parameters);
+
+        } catch (FHIRSearchException se) {
+            throw se;
+        } catch (Exception e) {
+            throw SearchExceptionUtil.buildNewParseParametersException(e);
```

Comment:
Can we pack in some additional detail in the exception?  e.g. what's the invalid combo?

Reply:
actually in this branch it's only going to be config related exceptions.

Reply:
Correct.